### DISCUSSION TITLE
ToolbarMenu/ToolbarListbox examples

### DISCUSSION THREAD

**Discussion by mvdiener** [graphql/graphiql#3002] [Q&A / Support] [answered]
Hello, I would really like to utilize the toolbar menu and listbox as shown in this [PR's recording](https://github.com/graphql/graphiql/pull/2631). However, my many attempts to get this working have been completely fruitless, and scouring the code and documentation hasn't yielded anything I've been able to understand. My extremely limited React knowledge is probably not helping either. Can anyone provide any guidance to using these two items? Thank you!

**Comment by trkohler** [comment]
joining the request.

**Comment by trkohler** [comment, marked as answer]
this have worked out for me

```
<ToolbarListbox label="platform" button={
                  <ToolbarButton label="platform">
                  <GiPlatform />
                </ToolbarButton>
                }>
                
                  <ToolbarListbox.Option value="1">option 1</ToolbarListbox.Option>
                  <ToolbarListbox.Option value="2">option 2</ToolbarListbox.Option>
                  <ToolbarListbox.Option value="3">option 3</ToolbarListbox.Option>
                </ToolbarListbox>
```
